db/db_impl/db_impl.cc

```diff
@@ -714,12 +714,17 @@ Status DBImpl::CloseHelper() {
     write_buffer_manager_->RemoveDBFromQueue(wbm_stall_.get());
   }
 
+  IOStatus io_s = directories_.Close(IOOptions(), nullptr);
```

Comment:
nit: we try to add comment to literal argument like `directories_.Close(IOOptions(), nullptr /* dbg */);`